c/loop-simple/nested_4.c

```diff
@@ -0,0 +1,34 @@
+// This file is part of the SV-Benchmarks collection of verification tasks:
+// https://github.com/sosy-lab/sv-benchmarks
+//
+// This file was part of CPAchecker,
+// a tool for configurable software verification:
+// https://cpachecker.sosy-lab.org
+//
+// SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+void reach_error(){}
```

Comment:
**question (non-blocking)**:@dbeyer @tautschnig how to proceed with PRs that have this in them? If we merge them after PRs 1145-1147 then we might have tasks with empty body for `reach_error` again. Ideally we could just execute the perl script again to fix this? Would that script be idempotent or would it modify other tasks yet again?

Reply:
I would prefer to have new code to include the assert already.

Reply:
(But no big deal anyway, since those are just a few files, you can apply the perl script and see if it fits.)

Reply:
ok, I will take a look.

Reply:
Can you share your experience? Where your changes done by the perl script or did you have to do it manually? That information would help others to judge whether it makes sense using the perl script.

Reply:
At first I did not find the mentioned  perl script, and after Philipp mentioned a PR,
I found a bash+perl script in the commit message (!) of 07dd9f37eb679d61e609d9ed79226a61a0bf6279 .
As the script is not documented (and not easily readable or understandable),
I decided to edit the 7 files manually, i.e., copy-paste for 2 lines was quite simple.

If I would have more benchmark tasks to submit,
I would like to see a proper script in the main directory of the repository.

Reply:
I created https://github.com/sosy-lab/sv-benchmarks/issues/1150 that will hopefully spark some general discussion in that direction.